CHANGES.rst

```diff
@@ -1,6 +1,17 @@
 Changes
 =======
 
+Version 0.5.0 (2019-04-17)
+--------------------------
+
+- Makes workflow engine independent of celery and created on demand.
+- Replaces ``api_client`` module with centralised one from REANA-commons.
+- Introduces CVMFS mounts in job specifications.
+- Makes docker image slimmer by using ``python:3.6-slim``.
+- Centralises log level and log format configuration.
+- Adds ``workflow_uuid`` to ``job_request_body``.
+- Sets the same ``click`` and ``pytest`` requirements for all REANA components.
```

Comment:
The last phrase about click/pytest is perhaps unnecessary... Otherwise reformulate like:

- Implements several minor improvements such as standardising click/pytest/etc prerequisites across all REANA platform components.